website/index.html

```diff
@@ -775,7 +775,7 @@
           </div>
           <div class="d-flex justify-content-between">
             <div class="col-md-4 col-xs-4">
-              <p class="time"></p>
+              <p class="time">2 PM</p>
```

Comment:
Same thing as above. "Removing the 2 PM here makes the layout a little cleaner. We did this last semester so there aren't redundant times listed (2PM twice)"